### ISSUE TITLE
Installation broken in python 3.12

### ISSUE THREAD

**Issue by israelmcmc** [cositools/cosipy#127]
The issue is actually with astromodels

https://github.com/threeML/astromodels/issues/204

**Comment by ckarwin** [comment]
@israelmcmc Do we have a current workaround for this issue? Is it just to use a python version <3.12, or just not to use a version =3.12 specifically?

**Comment by israelmcmc** [comment]
Yeah, currently the workaround is to not use 3.12. And neither 3.11, also due to an issue related astromodels: https://github.com/threeML/astromodels/issues/201 .

**Comment by israelmcmc** [comment]
Astromodels updated the installation setup and it now works for me in all python version. 

Let's close this issue when the following changes are included in a release (they are currently only in the dev branch):
https://github.com/threeML/astromodels/pull/226

**Comment by augustus-thomas** [comment]
Closed in #436
